### DISCUSSION TITLE
Adjust Difficulty Multiplier for DT

### DISCUSSION THREAD

**Discussion by cuhoangdk** [ppy/osu#21963] [Ideas] [answered]
since no one answer my previous post, i decide to post this again 
I found a function to make Speed Increase more reliable to Difficulty Multiplier
`difficulty mutiplier=(0.12/0.5)*(speed increase -1)+1`
Here is some special point that people usually use 
![Screenshot 2022-12-31 102448](https://user-images.githubusercontent.com/104707960/210123408-e518a9d4-fcd8-4213-8cc1-6148fefd13f4.png)
x: Speed Increase
the right side: Difficulty Multiplier
i hope you will use this in next update

**Comment by bdach** [comment]
**Do NOT repost discussions if they haven't been responded to yet.** If you continue to do this we may revoke your permissions to do so.

We know that multipliers for custom rates are not in a final state. For now this does not matter until we stabilise the experimental website. Once that is done, there will be a wide discussion as to how custom rates are going to work with respect to score submission. Only after that can we begin to talk about multipliers.

**Reply by cuhoangdk** [reply]
ok, i will wait for this
thanks for the answer

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/21914#discussioncomment-4571448
